extensions/wikia/AdEngine/js/provider/gpt/adDetect.js

```diff
@@ -149,14 +149,16 @@ define('ext.wikia.adEngine.provider.gpt.adDetect', [
 		return 'inspect_iframe';
 	}
 
-	function onAdLoad(slotname, gptEvent, iframe, adCallback, noAdCallback, forcedAdType) {
+	function onAdLoad(slotName, gptEvent, iframe, adCallback, noAdCallback, forcedAdType) {
 
-		var adType = forcedAdType || getAdType(slotname, gptEvent, iframe),
+		var adType = forcedAdType || getAdType(slotName, gptEvent, iframe),
 			shouldPollForSuccess = false,
 			expectAsyncHop = false,
 			expectAsyncHopWithSlotName = false,
+			expectAsyncSuccessWithSlotName = false,
 			expectAsyncSuccess = false,
-			successTimer;
+			successTimer,
+			shortSlotName = slotName.replace(/^.*\/([^\/]*)$/, '$1');
```

Comment:
Everytime I see a regex I think: "Unit test!" ;)

![Unit test all the things!](http://emappetizer.com/wp-content/uploads/2015/04/unit-test.jpg)